src/uct/ib/base/ib_iface.c

```diff
@@ -423,7 +427,7 @@ void uct_ib_iface_address_pack(uct_ib_iface_t *iface, uct_ib_address_t *ib_addr)
     params.lid       = uct_ib_iface_port_attr(iface)->lid;
     params.roce_info = iface->gid_info.roce_info;
     /* to suppress gcc 4.3.4 warning */
```

Comment:
pls, remove the comment that's unrelated anymore

Reply:
comment is moved to next line (there were 2 lines noted in GCC)